composer.json

```
@@ -25,13 +25,13 @@
         "cocur/slugify": "^3.0 || ^4.0",
         "doctrine/doctrine-bundle": "^1.12.3 || ^2.0",
         "doctrine/persistence": "^1.3.4 || ^2.0",
-        "sonata-project/admin-bundle": "^3.99",
-        "sonata-project/block-bundle": "^3.20",
+        "sonata-project/admin-bundle": "^3.91.1",
```

```suggestion
        "sonata-project/admin-bundle": "^3.99",
```
This line shouldn't be reverted.